content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: OpenTelemetry
+description: This task shows you how to configure Envoy proxies to send access logs with OpenTelemetry collector.
+weight: 10
+keywords: [telemetry,logs]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+After completing this task, you understand how to have your application participate in [Envoy's access logging](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage) with [OpenTelemetry (gRPC) Access Log](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/open_telemetry/v3/logs_service.proto).
+In this example, the Envoy proxies send access logs to an [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector), which is configured to print the logs to standard output.
+The standard output of the OpenTelemetry collector can then be accessed via the `kubectl logs` command.
+
+{{< boilerplate before-you-begin-egress >}}
+
+{{< boilerplate start-httpbin-service >}}
+
+{{< boilerplate start-otel-collector-service >}}
+
+## Enable Envoy's access logging
+
+To enable access logging, use the [Telemetry API](/docs/tasks/observability/telemetry/).
+
+### Using Telemetry API
+
+The Telemetry API can be used to enable or disable access logs:
```

Comment:
```suggestion
```

Assuming this is the only supported method, we don't need the heading for the choice